src/io/ctrees_utils.c

```diff
@@ -112,27 +114,38 @@ int64_t read_locations(const char *filename, const int64_t ntrees, struct locati
             XASSERT(ntrees_found < ntrees, EXIT_FAILURE,
                     "ntrees=%"PRId64" should be less than ntrees_found=%"PRId64"\n",
                     ntrees, ntrees_found);
-            int nitems = sscanf(buffer, "%"SCNd64" %d %"SCNd64 "%s",
+            const int nitems = sscanf(buffer, "%"SCNd64" %d %"SCNd64 "%s",
```

Comment:
Explicit mention of `_32t` etc.

Reply:
I know your war with `int` but really can't go wage that war. It's one of those places where our efforts are better off fixing something else. I agree with your sentiment for all variables might potentially get written to disk